alchemical_analysis/parser_amber.py

```diff
@@ -624,6 +624,7 @@ def readDataAmber(P):
     # FIXME: compute maximum number of MBAR energy sections
     K = len(lvals)
     nsnapshots = [len(dvdl_all[clambda]) - start_from for clambda in lvals]
+    nsnapshots = [int(i*P.breaks) for i in nsnapshots]
```

Comment:
Won't this go out of range in some cases?

Reply:
@davidlmobley I don't think so, basically, it just update the nsnapthots by multiplying the numbers in the current nsnapshots list with the P.breaks (not P.fraction) value.